pkg/kubectl/cmd/drain_test.go

```diff
@@ -32,12 +32,13 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
-
+	"k8s.io/api/core/v1"
```

Comment:
`v1.Node` is a stable version and won't be changed, but `internal` type may change all the time, https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md#operational-overview , maybe this is just a nitpick, I'm ok if others have no objections. : )